pipeline/pipeline.yaml

```diff
@@ -0,0 +1,1023 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: build-and-deploy
+spec:
+  params:
+  - description: docker registry namespace.  Choose something unique.  If using the local OpenShift registry, make this the same as NAMESPACE.
+    name: docker-namespace
+    type: string
+  - description: namespace to deploy control plane to.  If nobody has deployed fybrik on this cluster before, choose fybrik-system.
+    name: NAMESPACE
+    type: string
+  - default: kind-registry:5000
+    description: docker registry hostname.  This is where images will be pushed once built
+    name: docker-hostname
+    type: string
+  - default: docker.io
+    description: where to pull dockerhub images from.
+    name: dockerhub-hostname
+    type: string
+  - default: vascan  
+    description: namespace in us.icr.io where images will be pushed for VA scan
+    name: va-scan-namespace
+    type: string
+  - description: url of the git repo for the code of deployment
+    name: git-url
+    type: string
+    default: github.com/mesh-for-data/mesh-for-data.git
+  - default: pipeline
+    description: branch/revision to be used from repo of the code for deployment
+    name: git-revision
+    type: string
+  - default: "false"
+    description: deploy vault or no.  If deploying to the fybrik-system namespace, make this true.
+    name: deployVault
+    type: string
+  - default: "/workspace/source/mesh-for-data/third_party/vault/vault-single-cluster/vault-rbac.yaml"
```

Comment:
can the directory be made relative?

Reply:
This is referencing a location on persistent volume, so it should be consistent from install to install, but I want to verify whether or not this will need to change to 'fybrik' once merged from my repo to this repo